next.config.js

```diff
@@ -1,16 +1,29 @@
 /** @type {import('next').NextConfig} */
 module.exports = {
-  output: 'standalone',
+  output: "standalone",
   images: {
-    domains: [
-      'play.google.com',
-      'res.cloudinary.com'],
-  }
+    domains: ["play.google.com", "res.cloudinary.com"],
+  },
+  publicRuntimeConfig: {
+    BASE_URL: process.env.BASE_URL || "https://devme.bettersocial.org",
+    DOWNLOAD_BETTERSOCIAL_APP_URL:
+      process.env.DOWNLOAD_BETTERSOCIAL_APP_URL ||
+      "https://linkdev.bettersocial.org/download",
+    HUMAN_INTERNET_URL:
+      process.env.HUMAN_INTERNET_URL || "https://human-internet.org/",
+    DYNAMIC_LINK_DOMAIN:
+      process.env.DYNAMIC_LINK_DOMAIN || "https://devme.bettersocial.org",
+  },
+  serverRuntimeConfig: {
+    HUMAN_ID_GET_WEB_LOGIN_URL: process.env.HUMAN_ID_GET_WEB_LOGIN_URL,
+    HUMAN_ID_CLIENT_ID: process.env.HUMAN_ID_CLIENT_ID,
+    HUMAN_ID_CLIENT_SECRET: process.env.HUMAN_ID_CLIENT_SECRET,
+  },
   // webpackDevMiddleware: config => {
   //   config.watchOptions = {
   //     poll: 1000,
   //     aggregateTimeout: 300,
   //   }
   //   return config
   // },
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This commented out section of code related to `webpackDevMiddleware` should be removed if it's not being used. Commented out code can cause confusion for other developers who might wonder why it's there but not being used.

```diff
-    // webpackDevMiddleware: config => {
-    //   config.watchOptions = {
-    //     poll: 1000,
-    //     aggregateTimeout: 300,
-    //   }
-    //   return config
-    // },
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->